packages/node-client/lib/index.ts

```diff
@@ -834,11 +834,18 @@ export class Nango {
         };
 
         if (config.params) {
-            options.params = config.params;
-        }
-
-        if (config.paramsSerializer) {
-            options.paramsSerializer = config.paramsSerializer;
+            if (typeof config.params === 'string') {
+                if (url.includes('?')) {
+                    throw new Error('Can not set query params in endpoint and in params');
+                }
+                url = new URL(`${url}${config.params.startsWith('?') ? config.params : `?${config.params}`}`).href;
```

Comment:
this assumes url is a valid url and config.params is a valid query string. Is that always true?

Reply:
Both can be invalid I guess. This new URL will throw if invalid so that's covered. For query params, it's harder to check since almost anything can be a valid query param. 

Do you have a suggestion?

Reply:
my concern is: is this code behave the same as before if url or query params are invalid? if same behavior as axios I am fine with it

Reply:
it's slightly different on handling query params (otherwise this PR would not exist),
Axios is already sending an invalid url error so this part is similar